### PR TITLE
moved footer to bottom and put a border around sign in

### PR DESCRIPTION
**Pull request by jkasato**
moved footer to bottom and put a border around sign in

### CODE REVIEW

**Review by mrobertson-csumb — CHANGES_REQUESTED** [review]
The files in the bin directory should not have been added

**Comment by jkasato** [comment]
> The files in the bin directory should not have been added

How do I removed those changes and only include the css file that I added?

File: .gitignore [hunk 1]
```diff
@@ -2,6 +2,7 @@ target/
 !.mvn/wrapper/maven-wrapper.jar
 !**/src/main/**
 !**/src/test/**
+!**/bin/**
```
Comment (mrobertson-csumb) [inline .gitignore]: this line as written is an unignore, if it stays this way, `git add .` will re-add all of bin's contents.
it should simply be:
bin/

File: src/main/resources/public/css/styles.css [hunk 2]
```diff
@@ -66,33 +74,54 @@ ul.nav li a:visited,
 ul.nav li a:hover,
 ul.nav li a:active,
 ul.nav li.active a {
+	padding:0;
```
Comment (mrobertson-csumb) [inline src/main/resources/public/css/styles.css]: use a code formatter on the file before committing

File: src/main/resources/public/css/styles.css [hunk 3]
```diff
@@ -66,33 +74,54 @@ ul.nav li a:visited,
...
+    background: black;
+    text-align: center;
+    margin-block: 5%;
+    border: 1px solid;
+	font-size:0.5em;
+	color:white;
+    position: absolute;
+ 	bottom: 0;
+  	width: 100%;
+  	height: 3 rem; /* Footer height */
```
Comment (mrobertson-csumb) [inline src/main/resources/public/css/styles.css]: no space between scalar and unit

File: src/main/resources/public/css/styles.css [hunk 4]
```diff
@@ -66,33 +74,54 @@ ul.nav li a:visited,
...
     background: black;
     text-align: center;
     padding-top: 1%;
     padding-bottom: 1%;
     margin-block: 5%;
     border: 1px solid;
+}*/
+
+
+#footer-id {
```
Comment (mrobertson-csumb) [inline src/main/resources/public/css/styles.css:90]: footer is unreadably small when rendered on the page.
footer is not anchored at the very bottom of the page, there is a gap about 1.5x the height of the footer.

**Review by mrobertson-csumb — CHANGES_REQUESTED** [review]
Needs minor changes.
See inline comments.

File: src/main/resources/public/css/styles.css [hunk 5]
```diff
@@ -1,130 +1,155 @@
...
+	background: black;
+	text-align: center;
+	font-size: 0.6em;
+	color: white;
+	position: absolute;
+	bottom: 0px;
+	height: 3rem; width : 100%;
+	border: 3px solid black;
+	width: 100%;
```
Comment (mrobertson-csumb) [inline src/main/resources/public/css/styles.css]: width already defined above.

File: src/main/resources/public/css/styles.css [hunk 6]
```diff
@@ -1,130 +1,155 @@
...
     text-align: center;
     padding-top: 1%;
     padding-bottom: 1%;
     margin-block: 5%;
     border: 1px solid;
+}*/
+#footer-id {
+	background: black;
+	text-align: center;
+	font-size: 0.6em;
```
Comment (mrobertson-csumb) [inline src/main/resources/public/css/styles.css]: 0.8em is the smallest that looks OK on my browser.

**Review by mrobertson-csumb — CHANGES_REQUESTED** [review]

**Review by mrobertson-csumb — APPROVED** [review]

**Pull request merged**